pkgs/development/python-modules/duckdb/default.nix

```
@@ -62,7 +65,12 @@ buildPythonPackage rec {
   ];
 
   # test flags from .github/workflows/Python.yml
-  pytestFlagsArray = [ "--verbose" ] ++ lib.optionals stdenv.hostPlatform.isDarwin [ "tests/fast" ];
+  pytestFlagsArray =
+    [ "--verbose" ]
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [ "tests/fast" ]
+    # the `tests` flag here prevents sqllogic tests from running;
+    # these could be run in a custom check hook if desired
+    ++ lib.optionals (!stdenv.hostPlatform.isDarwin) [ "tests" ];
```

```suggestion
  pytestFlagsArray = [ 
    "--verbose"
    "tests/fast" 
  ];
```

And add to the end of `disabledTestPaths`:
```sh

    # depends on missing fixture
    "test_query_progress"
] ++ lib.optionals stdenv.hostPlatform.isDarwin [ "tests/fast" ];
```

Rationale: Unit tests eat quite a bit of the build system's time and the fast tests are enough to assure we built this correctly. It seems strange to switch whole test sets based on platform.